model-setup-examples/basic-elevon.md

```diff
@@ -6,17 +6,27 @@ You usually see this sort of configuration on tailless aircraft.
 
 A good example of this is an aircraft known asa "flying wing".
 
-Here is what basic eleven set up might look like on your radio, we are using the default AETR output.
+![](/assets/flying-wing.jpg)
+
+As you can see from the picture above, the airleron and elevator surfaces reside in the same physical control surface.
```

Comment:
and the plane uses only one servo per control surface. 

Better to explicit about this, there are wonky configuration with two servos that allow flexing the flap.